.prettierrc

```
@@ -1,5 +1,5 @@
 {
-    "printWidth": 120,
+    "printWidth": 80,
```

arrow function の括弧についても、定義していただければ！（これであじよしさん環境とのdiffがなくなったはず）

```suggestion
    "arrowParens": "avoid",
    "printWidth": 80,
```